genome_designer/utils/data_export_util.py

```diff
@@ -119,7 +119,7 @@ def _add_key_value(data_dict):
 
 
 def export_var_dict_list_as_vcf(var_dict_list, vcf_dest_path_or_filehandle,
-        sample_alignment):
+        sample_alignment, caller_string='VARIANT_DICTIONARY_EXPORT'):
```

Comment:
Whatever you choose to rename the key, I think the default value here `VARIANT_DICTIONARY_EXPORT` also needs a rename. You use `GRAPH_WALK` and `COV_DETECT_DELETION` (or `COVERAGE` if you like my suggestion above).

Alternatively, rather than caller_string being an optional argument, you could make it required. It looks like all clients of this functions pass this argument. Is there a situation where a default value would make sense?